modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -61,58 +55,63 @@ Certificate files are typically stored as Base64 files, with extensions such as
 $ {orch-cli} create configmap custom-certs --from-file=__<bundle-file-path>__ -n={prod-namespace}
 ----
 +
-Add another `-from-file=_<bundle-file-path>_` flag to apply more than one bundle. Otherwise, create another ConfigMap.
+To apply more than one bundle, add another `-from-file=_<bundle-file-path>_`. Otherwise, create another ConfigMap. 
+// Wouldn't "alternatively" make more sense than "otherwise"? 
 
-. Label created ConfigMaps with both `app.kubernetes.io/part-of=che.eclipse.org` and `app.kubernetes.io/component=ca-bundle` labels:
+. Label created ConfigMaps with the `app.kubernetes.io/part-of=che.eclipse.org` and `app.kubernetes.io/component=ca-bundle` labels:
 +
 [subs="+attributes,+quotes"]
 ----
 $ {orch-cli} label configmap custom-certs app.kubernetes.io/part-of=che.eclipse.org app.kubernetes.io/component=ca-bundle -n <{project-context}-namespace-name>
 ----
 
-. Deploy {prod-short} if it has not been deployed before. Otherwise wait until the rollout of {prod-short} components finishes. If there are running workspaces, they should be restarted for the changes to take effect.
+. Deploy {prod-short} if it hasn't been deployed before. Otherwise wait until the rollout of {prod-short} components finishes. Restart running workspaces for the changes to take effect.
+
+== Troubleshooting imported certificate issues
+
+If issues occur after adding the certificates, verify the below specified values at a {prod-short} installation and workspace level.
 
-== Verification at the {prod-short} installation level
 
-When something does not work as expected after adding the certificates, here is a list of things to verify:
+.Verifying imported certificates at the {prod-short} installation level
 
-- In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/understanding/olm-what-operators-are.html[Operator] deployment, the namespace where the `CheCluster` is located contains labeled ConfigMaps with the right content:
+* In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/understanding/olm-what-operators-are.html[Operator] deployment, the namespace where the `CheCluster` is located contains labeled ConfigMaps with the right content:
```

Comment:
Again, omit links and the peer review note/question. 

``In case of a {prod-short} Operator deployment, the namespace where the `CheCluster` is located contains labeled ConfigMaps with the correct content:`

Note, I think for word choice `correct` is better since `right` could mean direction, not accuracy.